nucypher/blockchain/eth/sol/source/contracts/Issuer.sol

```diff
@@ -173,7 +173,8 @@ contract Issuer is Upgradeable {
         return totalSupply - Math.max(currentSupply1, currentSupply2);
     }
 
-    function verifyState(address _testTarget) public onlyWhileUpgrading {
+    function verifyState(address _testTarget) public {
```

Comment:
Why does the `onlyWhileUpgrading` modifier get dropped?

Reply:
Because next line is calling `verifyState` from parent `Upgradeable` and there this modifier. Did to avoid repetition and increasing of code size